scss/_variables.scss

```
@@ -1614,15 +1614,21 @@ $pagination-transition:             $transition-focus !default;
 // Boosted mod: no $pagination-border-radius-lg
 
 // Boosted mod
-$pagination-padding-end:            1.125rem !default;
 $pagination-icon:                   var(--#{$prefix}chevron-icon) !default;
-$pagination-icon-size:              subtract($spacer * 2, calc(var(--#{$prefix}border-width) * 2)) !default; // stylelint-disable-line function-disallowed-list
+$pagination-icon-min-width:         .625rem !default;
+$pagination-icon-size:              1.125rem !default;
 $pagination-icon-width:             add(.5rem, 1px) !default;
 $pagination-icon-height:            subtract(1rem, 1px) !default;
+$pagination-gap:                    10px !default;
 
 $pagination-active-item-bg:         $primary !default;
 $pagination-active-item-color:      color-contrast($pagination-active-item-bg) !default;
 $pagination-active-item-border:     $pagination-active-item-bg !default;
+
+$pagination-item-max-width:         $spacer * 2 !default;
```

```suggestion
$pagination-item-max-width:         $spacer * .5 !default;
```
Big numbers look weird atm